demo/utils/definition/table-definition/table-definition.js

```diff
@@ -39,39 +39,49 @@ function buildRows() {
 }`;
 
   definition.propTypes = assign({}, definition.propTypes, {
-    currentPage: "String",
-    filter: "Object",
-    highlightable: "Boolean",
-    onChange: "Function",
-    onHighlight: "Function",
-    onPageSizeChange: "Function",
-    onSelect: "Function",
-    pageSize: "String",
-    pageSizeSelectionOptions: "Object",
-    paginate: "Boolean",
-    selectable: "Boolean",
-    showPageSizeSelection: "Boolean",
-    shrink: "Boolean",
-    tbody: "Node",
-    thead: "Node",
-    totalRecords: "String"
+    actions: 'Array',
+    currentPage: 'String',
+    children: 'Node',
+    className: 'String',
+    filter: 'Object',
+    highlightable: 'Boolean',
+    onChange: 'Function',
+    onHighlight: 'Function',
+    onPageSizeChange: 'Function',
+    onSelect: 'Function',
+    pageSize: 'String',
+    pageSizeSelectionOptions: 'Object',
+    paginate: 'Boolean',
+    selectable: 'Boolean',
+    showPageSizeSelection: 'Boolean',
+    shrink: 'Boolean',
+    sortOrder: 'String',
+    sortedColumn: 'String',
+    tbody: 'Node',
+    thead: 'Node',
+    totalRecords: 'String'
   });
   definition.propDescriptions = assign({}, definition.propDescriptions, {
-    currentPage: "Controls the current page number of a paginated data set.",
+    actions: 'The actions to display in the toolbar',
+    currentPage: 'Controls the current page number of a paginated data set.',
+    children: 'This component supports children.',
+    className: 'Classes to apply to the component.',
     filter: "An object of filtered data. Each key in the object should match with the key of one of the table's columns.",
-    highlightable: "Makes each row clickable/highlightable. Works well with the onHighlight callback.",
+    highlightable: 'Makes each row clickable/highlightable. Works well with the onHighlight callback.',
     onChange: "Triggered whenever any of the table's display is updated - for example if sort order changes, pagination changes or the filter changes. It will trigger your callback with all of the information required to manually filter/sort your data in your external data source.",
-    onHighlight: "Triggered when a row is highlighted via the highlightable prop. It will provide you with the row index.",
```

Comment:
I can revert the changes in this file as they aren't actually picked up by `gulp test`. This file was actually changed by my editor's eslint plugin when I hit save.